Get empty series working

irwalker opened the pull request:
Don't sample x-axis values unless there are values there to sample. Also
don't try and sanitise empty datasets, just return the empty dataset.
The underlying library supports empty sets.

This works for both plain empty series, and series with a label but no data.

E.g.

```
<%= column_chart([], {
title: 'My graph',
xtitle: 'Horizontal Series',
ytitle: 'Vertical Series',
theme: 'mytheme'
}) %>
```

and

```
<%= column_chart(
  [
    {
      name: 'Series A',
      data: []
    },
  ],
title: 'My graph',
xtitle: 'Horizontal Series',
ytitle: 'Vertical Series',
theme: 'mytheme'
}) %>
```

Would now both be valid.

This is a fix for bug 41

styd commented on lib/apexcharts/version.rb:
```diff
@@ -1,5 +1,5 @@
 
 module ApexCharts
-  VERSION = '0.1.6'
+  VERSION = '0.1.7'
```
No need to update the version just yet. I'm assigning myself as the release manager. So, let me decide when to bump up the version. Usually, I bump the version up after adding a few features and tested the them on my app a few times.

irwalker replied:
No worries I've removed that.

irwalker commented:
> This looks great. :-) But let me test it on my app once I get home. I'm still at work. While I test, can you revert back the version number? And maybe add a "Fixed" header for issue you are fixing on changelog. Thanks a lot.

Thanks; I've updated the changelog and reverted the version number. Happy testing

styd commented:
Thank you for your contribution.
